docs/src/main/sphinx/connector/hive.rst

```diff
@@ -622,6 +622,23 @@ configuration file ``hive-site.xml`` and restart the metastore service:
         <value>org.apache.hadoop.hive.metastore.SerDeStorageSchemaReader</value>
     </property>
 
+File-based metastore
+^^^^^^^^^^^^^^^^^^^^
+
+In order to use a file-based metastore, you must configure the metastore with:
```

Comment:
Needs some more introductory text explaining what this is. Something like "The connector supports use of an experimental, file-based metastore. The metastore must be accessible to Trino as a file in S3 or HDFS, and is specified in the `hive.metastore.catalog.dir` catalog configuration property.